releasenotes/notes/remove_probability_distributions-d30bd77f0f2b9570.yaml

```diff
@@ -0,0 +1,6 @@
+---
+upgrade:
+  - |
+    The module :mod:`qiskit.circuit.library.probability_distributions` has been removed and no longer exist as per the deprecation notice from 0.17.0
```

Comment:
This cross-ref won't link after the removal, so it's better just listing it with double backticks ` ``qiskit.<...>`` `.  It's probably best to list the three individual classes as well, because users were probably importing them from `qiskit.circuit.library` - they probably won't recognise the internal directory name.

Reply:
good point. `:mod:` removed in 53f2b7586